app/models/game.rb

```diff
@@ -9,6 +9,10 @@ class Game < ApplicationRecord
   has_many :clothing_items, dependent: :destroy
   has_many :ingredients, dependent: :destroy
   has_many :jewelry_items, dependent: :destroy
+  has_many :misc_items, dependent: :destroy
+  has_many :potions, dependent: :destroy
+  has_many :properties, dependent: :destroy
+  has_many :staves, dependent: :destroy
```

Comment:
Had to define all these associations - I'd neglected to do so when I introduced these models.